gemmbench/gemm_utils.py

```diff
@@ -61,40 +59,54 @@ def generate_mlir(config: GemmConfig):
     K = config.K
     M = config.M
     N = config.N
-    dtype = config.dtype
+    operand_element_type = config.operand_element_type
+    acc_element_type = config.accumulator_element_type
+    result_element_type = config.result_element_type
+    assert not operand_element_type.startswith('i'), "Integer types not supported yet"
```

Comment:
What's the reason integer types not supported? (would it not just work with this i8 operand, i32 accumulator, i8 result type?)

Reply:
Well they are not supported elsewhere either -- I didn't want to plumb through int8 support in the same PR